dotcom-rendering/src/lib/commercial-bundle.ts

```diff
@@ -0,0 +1,3 @@
+export const decideCommercialBundleUrl = (
+	commercialBundleUrl: string,
+): string => process.env.COMMERCIAL_BUNDLE_URL ?? commercialBundleUrl;
```

Comment:
Bit of a nitpick but could this code just be inlined?

It's true that this would create duplication but given this is just one line of code maybe the benefit of having the code be more readable (I don't need to open a new file to understand what's happening) is worth the price? I really don't have a problem with this level of duplication.

Reply:
I think that's a fair point, will make the change. Thanks!